accord-core/src/main/java/accord/topology/Shard.java

```diff
@@ -116,6 +116,8 @@ public String toString(boolean extendedInfo)
                     sb.append('f');
             }
             sb.append(')');
+            if (!joining.isEmpty())
```

Comment:
unrelated but its been useful to see how we handle TCM transitions... during host replacements we go into "joining" but logically we are "leaving" =D

Reply:
This is probably a bug and we can just change it?

Reply:
spoke with @bdeggleston and this is really `non-readable` in Cassandra... so just a case of bad naming.  Right now this is dead code so it doesn't impact anything, but it is something we should flesh out... we spoke before that we have an issue with this case as we will try to read from non-readable nodes and have to fall back... this adds latency to user requests that could be avoided by leveraging this.